ddd-koans/EduTools and basics/EduTools/task.html

```diff
@@ -0,0 +1,28 @@
+<html>
+<h1>EduTools plugin</h1>
+<h3>Plugin</h3>
+<p>
+    <a href="https://www.jetbrains.com/help/education/educational-products.html">Jetbrain's Educational Products</a> are
+    set of tools that can be used for teaching and learning programming in form of various different tasks.
+    <br><br>
+    To start learning one needs the EduTools plugin Thanks to that plugin, many courses
+    can be browsed and run in the Intellij IDE itself.
+    <br><br>
+</p>
+
+<h3>Working with the course</h3>
+<p>
+    There are three main parts of the Intellij's window that you will interact with:
+<ul>
+    <li>Course structure on the left</li>
+    <li>Code editor in the middle</li>
+    <li>Task description on the right</li>
```

Comment:
worth adding something about `Run` and `Next` buttons as it's the first time when the user needs to use it